app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```
@@ -814,19 +869,21 @@ $chevron-indent-spacing: 7px;
 }
 
 // Dropdown menu footer links.
-.gem-c-layout-super-navigation-header__navigation-second-footer {
-  border-top: 1px solid govuk-colour("mid-grey");
+.gem-c-layout-super-navigation-header__navigation-second-footer-break {
+  @include govuk-media-query($until: "desktop") {
+    display: none;
+  }
 }
 
 .gem-c-layout-super-navigation-header__navigation-second-footer-list {
   list-style: none;
-  padding-bottom: govuk-spacing(8);
-  padding-top: govuk-spacing(4);
-
-  @include govuk-grid-column($width: "two-thirds", $float: right, $at: "desktop");
+  padding: 0 0 govuk-spacing(8) 0;
```

Another potential tidy up:

```suggestion
  padding: 0 0 govuk-spacing(8);
```

I also wonder just in this case if this could be a bit of a nicer read if it was split out into separate lines:

```suggestion
  padding: 0;
  padding-bottom: govuk-spacing(8);
```